_includes/devdoc/ref_p2p_networking.md

```diff
@@ -693,102 +693,9 @@ d91f4854 ........................... Epoch time: 1414012889
 {% autocrossref %}
 
 *Added in protocol version 311.*
+*Removed in protocol version 70013 and released in Bitcoin Core v0.13.0*
```

Comment:
I think that's reasonable.

As a nitpick, could you remove the `v` in `v0.13.0`?  That will allow the auto-linker to link `Bitcoin Core 0.13.0` to the release notes for that version as it does in the protocol versions table:

![2017-05-10-05_17_58_254867513](https://cloud.githubusercontent.com/assets/61096/25891725/19909526-3540-11e7-93a0-05d4770f8d9e.png)